examples/Perl/googlevoice.pl

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/perl
+#
+# Uses snowboy to wake up and then collect audio to send to Google Speech API for further recognition
+# Takes Personal Model as a first argument
+
+use lib 'Snowboy';
+
+use Audio::PortAudio;
+use Data::Dumper;
+use Math::Round qw/round/;
+use LWP::UserAgent;
+use JSON;
+use Statistics::Basic qw(:all);
+use Time::HiRes qw(gettimeofday tv_interval);
+use IO::Handle;
+
+use Snowboy;
+
+die "Usage: $0 <model.pmdl>" 
```

Comment:
Yes I think it's OK. It's a universal model so it is supposed to work for everyone.